airflow/contrib/auth/backends/ldap_auth.py

```diff
@@ -157,8 +180,11 @@ def login(self, request):
         session.close()
 
         return redirect(request.args.get("next") or url_for("admin.index"))
-    except AuthenticationError:
-        flash("Incorrect login details")
+    except (LdapException, AuthenticationError) as e:
+        if e:
+            flash(e, "error")
```

Comment:
I do not believe so, but let me login to my work machine and test out this theory.

IIRC, there was no message in e.. but I could be very wrong (or I had a bad rebase)